src/DssDirectDepositAaveDai.sol

```diff
@@ -216,25 +224,53 @@ contract DssDirectDepositAaveDai {
 
         emit File(what, data);
     }
+
     function file(bytes32 what, address data) external auth {
+        require(vat.live() == 1, "DssDirectDepositAaveDai/no-file-during-shutdown");
```

Comment:
We don't need to block changing this during GS. This is for Aave rewards and has nothing to do with GS.

Reply:
Well we need to assume a malicious governance. We do not want they have access to change the `king`, however is quite probably the `king` will be the pause proxy, so they will still get the rewards.
Anyway I think is more about a good practice to block all the authed functions if the `vat` is caged.
But not feeling too strong about this specific case either (and we haven't followed that pattern in other contracts either).

Reply:
Right, fair point about attack. This is a pretty edge case that in practise will likely not amount to a large amount of funds since they are likely already sent and this thing will probably be closed out mostly during ES. I'm fine if you want to leave as-is.